stories/components/form/ArrayList.stories.mdx

```diff
@@ -0,0 +1,68 @@
+import { Meta, Canvas, Story, ArgsTable, Source } from '@storybook/addon-docs';
+import ArrayList from '@/pkg/rancher-components/src/components/Form/ArrayList/ArrayList';
+import { _EDIT, _VIEW } from "@shell/config/query-params";
+
+export const Template = (args, { argTypes }) => ({
+  components: { ArrayList },
+  props:      Object.keys(argTypes),
+  template:   '<ArrayList v-bind="$props" />',
+})
+
+<Meta 
+  title="Components/Form/ArrayList"
+  decorators={[]}
+/>
+
+# ArrayList
+The ArrayList is used to input a list of strings. The list is of variable length.
+
+## Default Empty ArrayList (Button only)
+<Canvas>
+  <Story
+    name="Empty"
+    args={{
```

Comment:
We can rely on default settings and replace this with `args` without brackets, as it still needs to define attribute.
`value` and `mode` are not needed.

Reply:
I have removed the args prop